code for careers dropdown and chatbot card

TharunEmuri opened the pull request.

medwut commented:
Tharun this is not right i think your whole pr is wrong. sorry it should not have 1 million plus changes .  Contanct Mentor or Team leader regarding this.

TharunEmuri commented:
intially i created pull request for the changes and modification i made , later i knew that there are lots of changes so i fixed every thing and i created a final pull reequest , in which i had added the code for chat bot and carrers dropdown menu , so the latest i.e and last pull request i made  is the right one , could you please check it once and confirm me again

uzairzafarr commented:
Hi Tharun, 
I think the changes you add is not correct, Career dropdown is not working and it also disturbing the home page. (Snapshot attached)

Please review and make changes asap. 
![image](https://github.com/user-attachments/assets/1481f1b3-fec1-4d5f-8a85-5956a01053f3)


![image](https://github.com/user-attachments/assets/a08cbf4d-0492-4b20-bf58-1626cf667a21)
